handbook/technical-overview/devops/dockerizing.md

```diff
@@ -20,6 +20,20 @@ That's it.
 
 When you push this change to `main` your Docker image will get built and pushed to our private central Docker registry
 
+## Can I create a secret in the aws parameters store for services to consume?
+
+Yes you can!
+
+You can run the `create-secret` script and you will be prompted for input.
+
+```bash
+./scripts/create-secret
+```
+
+You will be prompted for a _secret name_ that will be added to the `/k8s/` secrets namespace and a _secret value_. The length of the _secret name_ should be from 6-16 characters long. Only alphanumeric characters and dash (-) is allowed.
```

Comment:
One can create secrets like `/k8s/part1/part2/partN`, right?

Reply:
I think 16 is way too short for a secret name, maybe put no limit on it ;)

Reply:
good catch!

No they can't, I will need to fix the regex to allow `/` and maybe I should increase the max allowed chars

Reply:
secret names ending with `/` are now also rejected.

Reply:
missed the second comment, fixing now.